server/app/views/questiontypes/AddressQuestionRenderer.java

```diff
@@ -89,7 +88,7 @@ protected DivTag renderTag(
                 validationErrors.getOrDefault(addressQuestion.getZipPath(), ImmutableSet.of()))
             .addReferenceClass(ReferenceClasses.ADDRESS_ZIP);
 
-    if (hasQuestionErrors) {
+    if (!validationErrors.isEmpty()) {
```

Comment:
I was hoping to push this change further into `FieldWithLabel`, so that we didn't have to manage `FieldWithLabel` 's `aria-invalid` at this level.

However, we currently only pass the field level errors to `FieldWithLabel` (and support 3 different types - [link](https://github.com/civiform/civiform/blob/9a0ddd0468c22e09eb98b637d77efeac2aa9c1ac/server/app/views/components/FieldWithLabel.java#L276-L301)), and it felt a bit messy to try to expand it to pass in the whole error map.